src/language_features/hover.rs

```diff
@@ -69,7 +69,7 @@ pub fn editor_hover(
                         Some(format!(
                             "• {{{}}}{}{{{}}}",
                             face,
-                            escape_brace(x.message.trim())
+                            escape_kakoune_markup(x.message.trim())
```

Comment:
since wow, that's embarassing, thanks!

I renamed it to "escape_kakoune_markup" for consistency with other `_kakoune` functions